en/advanced_config/tuning_the_ecl_ekf.md

```diff
@@ -27,16 +30,24 @@ A complementary filter is used to propagate the states forward from the 'fusion
 The time constant for this filter is controlled by the [EKF2_TAU_VEL](../advanced_config/parameter_reference.md#EKF2_TAU_VEL) and [EKF2_TAU_POS](../advanced_config/parameter_reference.md#EKF2_TAU_POS) parameters.
 
 ::: info
-The 'fusion time horizon' delay and length of the buffers is determined by the largest of the `EKF2_*_DELAY` parameters.
-If a sensor is not being used, it is recommended to set its time delay to zero.
+The 'fusion time horizon' delay and length of the buffers is determined by [EKF2_DELAY_MAX](../advanced_config/parameter_reference.md#EKF2_DELAY_MAX).
+This value should be at least as large as the longest delay `EKF2\_\*\_DELAY`.
 Reducing the 'fusion time horizon' delay reduces errors in the complementary filter used to propagate states forward to current time.
 :::
 
+The EKF uses the IMU data for state prediction only. IMU data is not used as an observation in the EKF derivation.
+The algebraic equations for the covariance prediction and measurement jacobians are derived using [SymForce](https://symforce.org/) and can be found here: [Symbolic Derivation](https://github.com/PX4/PX4-Autopilot/blob/main/src/modules/ekf2/EKF/python/ekf_derivation/derivation.py).
+Covariance update is done using the Joseph Stabilized form to improve numerical stability and allow conditional update of independent states.
```

Comment:
Would a link be helpful or will all readers know what this is or be able to search? https://en.wikipedia.org/wiki/Kalman_filter#Deriving_the_posteriori_estimate_covariance_matrix

Reply:
Yes, good idea, we could link that wikipedia section